SCons/Environment.xml

```diff
@@ -1033,7 +1033,7 @@ either as separate arguments to the
 &f-Clean;
 method, or as a list.
 &f-Clean;
-will also accept the return value of any of the &consenv;
+will also accept the return value of the &consenv;
```

Comment:
A similar change is also in #4648

Reply:
Feel free to merge #4648 first, I'm OK to resolve the conflicts.